bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v18/V18PsbtRpc.scala

```diff
@@ -0,0 +1,32 @@
+package org.bitcoins.rpc.client.v18
+
+import org.bitcoins.rpc.client.common.Client
+import org.bitcoins.rpc.jsonmodels.AnalyzePsbtResult
+import org.bitcoins.rpc.serializers.JsonSerializers._
+import play.api.libs.json._
+
+import scala.concurrent.Future
+
+trait V18PsbtRpc {
+  self: Client =>
+
+  /**
+    * Set of utilities to analyze, join, and update existing PSBTs
+    * @see [[https://bitcoincore.org/en/doc/0.18.0/rpc/rawtransactions/analyzepsbt/]]
+    * @see [[https://bitcoincore.org/en/doc/0.18.0/rpc/rawtransactions/joinpsbts/]]
+    * @see [[https://bitcoincore.org/en/doc/0.18.0/rpc/rawtransactions/utxoupdatepsbt/]]
+    */
+
+  def analyzePsbt(psbt: String): Future[AnalyzePsbtResult] = {
```

Comment:
I could be wrong but I was under the impression that @torkelrogstad implemented a type for PSBTs?

Reply:
same comment applies for the two defs below

Reply:
He did not implement a type for PSBTs it was in a list of his todo's he wanted to add but never got to it. I believe there is an open issue for it right now